### ISSUE TITLE
More tracks in the llc file

### ISSUE THREAD

**Issue by s2xnetx** [mifi/lossless-cut#1945]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

Sometimes I create multiple styles of chapters for export, it would be great to have multiple tracks with different cutsegments in the llc file and be able to select the active track via gui. This way we dont need to create more llc files for one video.

**Comment by mifi** [comment]
by "tracks", you don't mean anything related to the video/audio/subtitle tracks <img width="100" alt="Screenshot 2024-04-10 at 22 38 52" src="https://github.com/mifi/lossless-cut/assets/402547/75d7cf6b-bdd3-4fa4-960a-aecdeea16b63"> right?

**Comment by s2xnetx** [comment]
No. I mean that you can have multiple cutsegment tracks. Think about it like you have two or more llc files in one file. If you change the "track" it loads different cutsegments.

**Comment by s2xnetx** [comment]
Any news on this feature ?

**Comment by mifi** [comment]
No. it seems that you are the only person who asked for this feature so far
